conf/defaults.ini

```diff
@@ -82,6 +82,9 @@ max_idle_conn = 2
 # Max conn setting default is 0 (mean not set)
 max_open_conn =
 
+# Set to true to log the sql calls and execution times.
+debug =
```

Comment:
I think we can give this setting a better name. How about `log_sql_queries` or `log_queries`?

Reply:
Great plan, I like the more descriptive name. It is already in the database section so I think we can leave off the 'sql', as there wouldn't be any other 'queries' we are talking about.